devel/tao/Portfile

```diff
@@ -17,7 +17,7 @@ distname            ${name_package}-${version}
 categories          devel
 platforms           darwin
 license             BSD
-maintainers         {gmail.com:tlockhart1976 @lockhart} openmaintainer
+maintainers         {gmail.com:tlockhart1976 @lockhart @jwillemsen} openmaintainer
```

Comment:
The format here is not correct. You need to add yourself in a separate set of `{}` braces, with both your GitHub handle and preferred email address.